index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.typekit.net/uzf0rar.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <title>Reneweable Energy Conference</title>
+</head>
+<body>
+    <section class="background">
+        <nav class ="desktopnav">
+            <div class="space"></div>
+            <ul class="socialmedia">
+                <li><a href="#"><i class=" fa fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class=" fa fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class ="desktop2nav">
+            <div class="space2"><img src="image/logo-EHS.png" class="desktoplogo"></div>
```

Comment:
- [ ] You have done incredibly well by using img tag. However, you should ensure that the alt has a text. This is because alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see. Kindly ensure to make your page more accessible by implementing the alt text 👍 .

Reply:
- [ ] It is good _**HTML**_ and _**accessibility**_ practice to use the `alt` attribute when using the `img` tag. `Alt` attributes describe the appearance and the function of the _**image**_ on the webpage. The **_screen readers_** when parsing through your webpage capture the `alt` attribute and those who are visually impaired use screen readers to navigate webpages and websites. 

#### I noticed you have a lot of `img` tags without the alt attribute in both the index.html and about.html files. Please include the `alt` attributes in the `img` tags. 💻 👍